tests/src/test/scala/org/virtuslab/tests/Benchmark.scala

```diff
@@ -1,16 +1,11 @@
-package com.twitter.intellij.pants
+package org.virtuslab.tests
 
-import org.junit.Test
-import org.virtuslab.ideprobe.RunningIntelliJFixture
+import com.twitter.intellij.pants.Benchmarks
+import org.virtuslab.ideprobe.{IdeProbeFixture, RunningIntelliJFixture}
 import scala.concurrent.duration.Duration
 
-class ProjectOpenBenchmark extends PantsTestSuite {
-
-  @Test def pants(): Unit = benchmark("pants", openProjectWithPants)
-
-  @Test def bsp(): Unit = benchmark("bsp", openProjectWithBsp)
-
-  private def benchmark(configPrefix: String, openProject: RunningIntelliJFixture => Unit): Unit = {
+trait Benchmark extends IdeProbeFixture {
+  def benchmark(configPrefix: String, openProject: RunningIntelliJFixture => Unit): Unit = {
```

Comment:
Either keep the original name or if it is meant to be more general, rename `openProject` to something more general as well